src/script/hooks/useHideElement.ts

```diff
@@ -1,57 +1,59 @@
 import {useEffect} from 'react';
-import {throttle} from 'Util/util';
 
 const useHideElement = (element: HTMLElement, timeout: number, skipClass?: string) => {
   useEffect(() => {
-    let hide_timeout: number = undefined;
+    if (!element) {
+      return undefined;
+    }
+
+    let hideTimeout: number;
     let isMouseIn: boolean = false;
 
-    const effect = () => {
-      if (!element) {
+    const onMouseEnter = () => {
+      isMouseIn = true;
+      element.classList.remove('hide-controls');
+    };
+
+    const onMouseLeave = () => {
+      isMouseIn = false;
+      element.classList.add('hide-controls');
+    };
+
+    const startTimer = () => {
+      hideTimeout = window.setTimeout(() => {
+        element.classList.add('hide-controls');
```

Comment:
The `'hide-controls'` string is used 5x in this code. I suggest adding a `const` with that name and reusing this const.